### ISSUE TITLE
Dependency Conflict with WebRTC

### ISSUE THREAD

**Issue by callum-code** [twilio/twilio-video-ios#207]
> Before filing an issue please check that the issue is not already addressed by the following:
>
>  * [Video Guides](https://www.twilio.com/docs/api/video)
>  * [Changelog](https://www.twilio.com/docs/api/video/changelog-twilio-video-ios-version-2x)
> 
> If this is an issue with the SDK itself, file it here. If this is an issue with the QuickStart apps, please use [video-quickstart-ios](https://github.com/twilio/video-quickstart-ios/issues).
>
> Please ensure that you are not sharing any
[Personally Identifiable Information(PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
or sensitive account information (API keys, credentials, etc.) when reporting an issue.

### Description

Heyo Twilio,

We've been going through our implementation of `WebRTC` and rename spacing all of the objective c types, similar to the process you'd kindly outlines [here](https://www.twilio.com/docs/video/ios-v4-using-twilio-video-side-side-webrtc).
When checking side by side with the version of WebRTC distributed on Cocoapods we were able to eliminate all of our dependency clashes. We appreciate that you'd done this as well. However it looks like you'd missed one. We are currently just getting this one dependency clash for `RTCDisplayLinkTimer`. I'm assuming you want this to be `TVIRTCDisplayLinkTimer`?

### Steps to Reproduce

1. add `TwilioVideo 3.8` and `WebRTC` to Podspec
2. bundle exec pod install
3. run and install to iOS device. Observe warning in output.

#### Code

WebRTC Source code that needs changed:

```
// RTCDisplayLinkTimer wraps a CADisplayLink and is set to fire every two screen
// refreshes, which should be 30fps. We wrap the display link in order to avoid
// a retain cycle since CADisplayLink takes a strong reference onto its target.
// The timer is paused by default.
@interface RTCDisplayLinkTimer : NSObject
```

and 

```
@implementation SRTCDisplayLinkTimer {
  CADisplayLink *_displayLink;
  void (^_timerHandler)(void);
}
```

(as well as all uses of the type)

You could probably use the `RTC_OBJC_TYPE` macro. I'm not sure why the Chromium team forgot to add that to this type. Maybe they'd missed it as well?

#### Expected Behavior

No dependency conflict with WebRTC

#### Actual Behavior

Dependency conflict with `RTCDisplayLinkTimer` type

#### Reproduces How Often

100%

#### Logs

```
objc[9588]: Class RTCDisplayLinkTimer is implemented in both /private/var/containers/Bundle/Application/{UUID}/{App}.app/Frameworks/WebRTC.framework/WebRTC (0x12f8a53c0) and /private/var/containers/Bundle/Application/{UUID}/{App}.app/Frameworks/TwilioVideo.framework/TwilioVideo (0x12ba60d08). One of the two will be used. Which one is undefined.
```

### Versions

`Xcode 12.5`

#### Video iOS SDK

`TwilioVideo 3.8`

#### Xcode

`Xcode 12.5`

#### iOS Version

`14.7.1`
(although our sdk minimum version is 11.0)

#### iOS Device

iPhone XS (I'm sure it would work on any)

**Comment by ceaglest** [comment]
Hi @callum-code,

> Video iOS SDK

> TwilioVideo 3.8

Does the conflict with RTCDisplayLinkTimer also occur in 4.5.0? I believe we changed to using the `RTC_OBJC_TYPE` macro internally for renaming in 4.x, but in 3.8 it was a different renaming solution applied at build time.

Best,
Chris

**Comment by callum-code** [comment]
Hi Chris, thanks for the quick reply. Updating to `4.5` resolved the warning for `RTCDisplayLinkTimer`, but introduced a new one for  `RTCWrappedEncodedImageBuffer`.

**Comment by ceaglest** [comment]
Roger. Our upcoming 4.6 release is based on a newer version of libwebrtc, let me get back to you. If it doesn't fix the collision we might be able to land a change upstream to make sure the macro is being used.

**Comment by callum-code** [comment]
Cheers!

**Comment by callum-code** [comment]
@ceaglest Had this made it into 4.6?

**Comment by mmartikainen** [comment]
This is fixed in Video iOS 5.1.1.
